deeppavlov/configs/morpho_tagger/UD2.0/ar/morpho_ar_train.json

```diff
@@ -0,0 +1,81 @@
+{
+  "dataset_reader": {
+    "name": "morphotagger_dataset_reader",
+    "data_path": "UD2.0_source",
+    "language": "ar", "data_types": ["train", "dev", "test"]
+  },
+  "dataset_iterator": {
+    "name": "morphotagger_dataset"
+  },
+  "chainer": {
+    "in": ["x"],
+    "in_y": ["y"],
+    "pipe": [
+      {
+        "id": "lowercase_preprocessor",
+        "name": "lowercase_preprocessor",
+        "in": ["x"],
+        "out": ["x_processed"]
```

Comment:
There is no explicit tokenizer component, tokenization in `lowercase_preprocessor` is weird.

Reply:
I have made tokenization more correct. However, I do not know how to make it explicit since tokenization is used only in `interact` mode, while the three others already work with tokenized sentences. It will require one more configuration file.

Reply:
You can look at https://github.com/deepmipt/DeepPavlov/blob/master/deeppavlov/models/tokenizers/lazy_tokenizer.py

Reply:
It is exactly what I am looking for, the only thing that embarrases me is that LazyTokenizer uses nltk.word_tokenize by default. Tokenization is not an issue for me, but in principle it is more correct to allow arbitrary tokenizer inside LazyTokenizer.